src/Grpc/HttpApi/Directory.Build.props

```diff
@@ -0,0 +1,15 @@
+<Project>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
+
+  <PropertyGroup>
+    <!-- Projects must be restored after App.Ref and App.Runtime layouts are complete. -->
+    <RequiresDelayedBuild>true</RequiresDelayedBuild>
+
+    <!-- Required because Swashbuckle.AspNetCore uses shared runtime -->
+    <UseAspNetCoreSharedRuntime>true</UseAspNetCoreSharedRuntime>
+
+    <!-- Workaround lack of Linux MUSL x64 support. -->
+    <ExcludeFromBuild
+        Condition=" '$(TargetOsName)' == 'linux-musl' and '$(TargetArchitecture)' == 'x64' ">true</ExcludeFromBuild>
```

Comment:
@JamesNK I'm not sure of the bug here but the tooling fails to start on Linux MUSL x64.

Reply:
Grpc.Tools has some native components. Might not support architecture?

Do the gRPC integration tests have the same issue? They also use Grpc.Tools.

Reply:
Not sure what you mean by "gRPC integration tests". If that's `InteropTests`, then no, they're fine because we don't run or build tests on Linux MUSL x64. Hit the problem here because the folder includes implementation projects and I hadn't got the exclusions in BuildAfterApp.csproj working right. (Yeah, I discovered a problem because something else was busted 😺)

Reply:
Put another way, I left this here as defence-in-depth and a reminder.